src/server/database-check.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-const config = require('../../.application-config.json');
+const config = process.env.NODE_ENV !== 'production'?
```

Comment:
do some formatting with this because it is really hard to read!

Reply:
maybe you can do something like

```js
if (process.env.NODE_ENV !== 'production') {
  const config = require('../../.application-config.json');
else {
  const config = {};
}
```
it is much more readable. also creating a function for this might be also good